doc/whats_new.rst

```diff
@@ -35,6 +35,8 @@ Changelog
     - Add reader for CTF data in :func:`mne.io.read_raw_ctf` by `Eric Larson`_
 
     - Add support for Brainvision v2 by `Teon Brooks`_
+    
+    - Improve speed of generalization across time decoding up to a factor of seven by `Jean-Remi King`_ and `Federico Raimondo`_ and `Denis Engemann`_.
```

Comment:
Can you add a ref to the relevant function? Makes it easier for people to look it up on the website. While you're in there if you could hit the line above about BV v2 that would help, too.